conda_forge_tick/auto_tick.py

```diff
@@ -336,6 +401,18 @@ def run(
         raise_commit_errors=migrator.allow_empty_commits and not rerender,
     )
 
+    if rerender:
+        # for version migrations, check solvable or automerge, we always raise rerender errors
+        suppress_errors = (
+            not is_version_migration
+            and not context.check_solvable
+            and not context.automerge
+        )
+
```

Comment:
The logic here appears wrong. I think we need the suggestion below. 

```suggestion
        # for version migrations, without check solvable or automerge, we can suppress rerender errors
        suppress_errors = (
            is_version_migration
            and not context.check_solvable
            and not context.automerge
        )

```

Reply:
jup, indeed